.circleci/config.yml

```diff
@@ -0,0 +1,210 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.8.0
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
+
+      - run:
+          name: Install Dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip3 install --upgrade pip
+            pip3 install -r requirements.txt
+
+      - save_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
+          paths:
+            - 'venv'
+
+      - run:
+          name: Install Resolver
+          command: |
+            . venv/bin/activate
+            pip3 install -e .
+
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
+
+  lint-and-unit-test:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.7.0
+
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+
+      - checkout
+
+      - restore_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
+
+      - run:
+          name: Install Dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip3 install --upgrade pip
+            pip3 install -r requirements.txt
+
+      - save_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
+          paths:
+            - 'venv'
+
+      - run:
+          name: Linting
+          command: |
+            . venv/bin/activate
+            make lint
+
+      - run:
+          name: Unit Test
+          command: |
+            . venv/bin/activate
+            mkdir test-reports
+            make test
+
+      - run:
+          name: Coverage
+          command: |
+            . venv/bin/activate
+            make coverage
+
+      - store_test_results:
+          path: test-reports
+          destination: test-reports
+
+      - store_test_results:
+          path: coverage.xml
+          destination: coverage-reports
+
+      - store_artifacts:
+          path: test-reports
+          destination: test-reports
+
+      - store_artifacts:
+          path: coverage.xml
+          destination: coverage-reports
+
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
+
+  deploy-test:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.7.0
```

Comment:
This too is a different version than the one at top.

Reply:
fixed